docs/data_resources.yaml

```diff
@@ -17,44 +17,23 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/data_resource'
+              oneOf:
+                - $ref: '#/components/schemas/DataService'
+                - $ref: '#/components/schemas/DataSet'
         required: true
       responses:
         '201':
           description: Data Resource Created
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/data_resource'
-              examples:
-                New Data Resource Citizen API:
-                  value:
-                    identifier: abcd1234
-                    type: DataService
-                    title: Citizen API
-                    description: Provides access to DWP citizen information to Other Government Departments and approved external agencies.
-                    keywords:
-                      - dwp
-                      - citizen
-                    theme:
-                      - https://www.data.gov.uk/search?filters%5Btopic%5D=Society
-                    license: https://opensource.org/license/isc-license-txt/
-                    version: 1
-                    accessRights: OPEN
-                    issued: 2022-05-10
-                    contactPoint:
-                      contactName: John Doe
-                      email: john.doe@example.com
-                    modified: 2023-01-06
-                    creator: department-for-work-pensions
-                    publisher: department-for-work-pensions
+          schema:
```

Comment:
I think you still need the content type for the response, no? And perhaps a description of what this mystery string response is 😄